service/handlers.go

```diff
@@ -67,6 +67,7 @@ func AuthHandler(h http.Handler) http.Handler {
 		}
 
 		context.Set(r, "token", token)
+		context.Set(r, "claims", claims)
```

Comment:
please use a constant here that is exported and can be used for getting the value too, and if you can make for above token to pls.